pact-core/Pact/Core/IR/Eval/CEK.hs

```diff
@@ -535,13 +543,13 @@ resumePact
   -> Maybe DefPactExec
   -> m (CEKEvalResult step b i m)
 resumePact i cont handler env crossChainContinuation = viewEvalEnv eeDefPactStep >>= \case
-  Nothing -> throwExecutionError i DefPactStepNotInEnvironment
+  Nothing -> throwExecutionError i DefPactStepNotInEnvironment    -- <- TODO apparently can't happen, `continuePact` ensures that
```

Comment:
Can't be triggered from REPL either, but a caller on chainweb's side could've done it.